tests/e2e/common.go

```diff
@@ -107,3 +116,33 @@ func printPodLogs(client clientgo.Interface, namespace, podName, containerName s
 	logBytes, _ := io.ReadAll(stream)
 	fmt.Printf("Printing Log for namespace=%q, pod=%q, container=%q:\n%s\n", namespace, podName, containerName, string(logBytes))
 }*/
+
```

Comment:
can you delete my commented out function above? (now that you are essentially doing almost the same thing)